pkg/AnewLibrary/Fields/Nodes/BooleanFieldNode.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2023 Atom54 LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+export const BooleanFieldNode = {
```

Comment:
Same question about ...Node.js files - i thought they'd be deprecated?

Reply:
This is just in progress.